src/types.db

```diff
@@ -148,6 +149,9 @@ node_octets             rx:DERIVE:0:U, tx:DERIVE:0:U
 node_rssi               value:GAUGE:0:255
 node_stat               value:DERIVE:0:U
 node_tx_rate            value:GAUGE:0:127
+nr_overcommit_hugepages value:DERIVE:0:U
+nr_hugepages            value:DERIVE:0:U
```

Comment:
So, we have done the following, I hope it is to your satisfaction:

- Added 1 new type, "hugepages"
- Gauge value, same limits as vmpage_number
- However, we have added 2 expected values: "free" and "used"
- We calculate "used" from the other metrics but we do not report the other metrics

I hope this is in line with what you requested in this and other comments.